### PR TITLE
Add sampling from copula

### PR DESCRIPTION
**Pull request by loikein**
This is (hopefully) the step 2 of [Conditional sampling](https://nbviewer.jupyter.org/github/OpenSourceEconomics/econsa/blob/master/docs/source/tutorials/conditional_sampling.ipynb).

Roadmap on Clickup: https://app.clickup.com/2509960/v/dc/2ck48-267/2ck48-141

### CODE REVIEW

**Comment by loikein** [comment]
Ok, this looks better… but still…

**Comment by loikein** [comment]
Sometimes I really need a flipping table emoticon or two. (╯°□°）╯︵ ┻━┻

File: econsa/tests/test_copula.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    dependent_ind = np.random.choice(full, replace=False, size=dependent_n)
+
+    for i in dependent_ind:
+        given_ind.remove(i)
+
+    means = np.random.uniform(-100, 100, dim)
+
+    # Draw new cov until is well-conditioned
+    while True:
+        cov = np.identity(dim) * np.random.uniform(0.05, 10)
```
Comment (peisenha) [inline econsa/tests/test_copula.py:34]: You can move this out of the loop, that matrix is always well-conditioned enough.

File: econsa/copula.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+This module contains functions to sample random input parameters from a Gaussian copula.
+"""
+import numpy as np
+from scipy.stats import multivariate_normal as multivariate_norm
+from scipy.stats import norm
+
+from econsa.sampling import cond_mvn
+
+
+def cond_gaussian_copula(cov, dependent_ind, given_ind, given_value_u):
```
Comment (peisenha) [inline econsa/copula.py:12]: Please add the ability to sample more than one deviate from the conditional distribution by adding an argument with a default value `size=1` and then use that in line 70 to draw multiple deviates.

File: .github/workflows/continuous-integration-workflow.yml [hunk 3]
```diff
@@ -30,32 +30,26 @@ jobs:
 
         # We want to make sure proper formatting of the notebooks.
         flake8-nb; [ $? -eq 0 ] || exit 1
-        black-nb .; [ $? -eq 0 ] || exit 1
+        black-nb --check .; [ $? -eq 0 ] || exit 1
 
         # We want to pull the submodules.
         # This can be removed once temfpy is live on conda.
```
Comment (peisenha) [inline .github/workflows/continuous-integration-workflow.yml:36]: This can go, right?

File: econsa/tests/test_copula.py (hunk 1 above)
Comment (loikein) [reply econsa/tests/test_copula.py:34]: Ok, cool!

File: .github/workflows/continuous-integration-workflow.yml (hunk 3 above)
Comment (loikein) [reply .github/workflows/continuous-integration-workflow.yml:36]: Er, no, do we not want to check black-nb? I actually copied this from your last fix branch…

**Pull request merged**